Packet++/src/HttpLayer.cpp

```diff
@@ -793,9 +793,9 @@ int HttpResponseFirstLine::getStatusCodeAsInt() const
 std::string HttpResponseFirstLine::getStatusCodeString() const
 {
 	std::string result;
-	int statusStringOffset = 13;
 	if (m_StatusCode != HttpResponseLayer::HttpStatusCodeUnknown)
 	{
+		int statusStringOffset = 13;

```

Comment:
I'm not sure I understand this change. Maybe we should turn this variable into a `const`?

Reply:
What I meant is that now we can move this back to line 796 and `cppcheck` should not give a warning?

Reply:
Oh sorry I understand now fixing